sdk/messaging/azservicebus/example_receiver_test.go

```diff
@@ -86,3 +86,30 @@ func ExampleReceiver_ReceiveMessages() {
 		fmt.Printf("Received and completed the message\n")
 	}
 }
+
+func ExampleReceiver_DeadLetterMessage() {
+	// DeadLetterMessage settles a message by moving it to the dead letter queue.
+	deadLetterReason := "exampleReason"
+	deadLetterErrorDescription := "exampleErrorDescription"
+
+	deadLetterOptions := &azservicebus.DeadLetterOptions{
+		Reason:           &deadLetterReason,
+		ErrorDescription: &deadLetterErrorDescription,
+	}
+
+	ctx, cancel := context.WithTimeout(context.TODO(), 60*time.Second)
+	defer cancel()
+  // How to create a dead letter receiver
+	// https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azservicebus/example_receiver_test.go#L25
+	messages, err := receiver.ReceiveMessages(ctx, 1, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	if len(messages) == 1 {
+		err := receiver.DeadLetterMessage(ctx, messages[0], deadLetterOptions)
```

Comment:
You _probably_ don't want to reuse `ctx` here since some of the time will be eaten by the ReceiveMessages() call, which could itself take time if no messages are available.

I would just cut the context part out entirely and just use context.TODO() in both calls.

Reply:
Apologizes for the delay, I'm turning back on PR notifications via email so I don't miss these in the future.